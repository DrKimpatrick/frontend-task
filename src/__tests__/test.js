import React from 'react';
import { shallow } from 'enzyme';
import Test from '../components/test';

describe('Test component', () => {
  it('Should render test componet', () => {
    const component = shallow(<Test />);
  
    expect(component).toMatchSnapshot();
  });
});