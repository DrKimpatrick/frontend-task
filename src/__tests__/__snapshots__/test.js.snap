// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component Should render test componet 1`] = `
<h1>
  Test.....
</h1>
`;
